<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDialog</class>
 <widget class="QDialog" name="ProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_lblA">
     <property name="text">
      <string>Project A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="_lblB">
     <property name="text">
      <string>Project B</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="prch::ProjectListWidget" name="_projectA"/>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" rowspan="4">
    <widget class="prch::ProjectListWidget" name="_projectB"/>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="_leftBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/img/left_arrow</normaloff>:/img/left_arrow</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="_rightBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="assets.qrc">
       <normaloff>:/img/right_arrow</normaloff>:/img/right_arrow</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>prch::ProjectListWidget</class>
   <extends>QListWidget</extends>
   <header>projectlistwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_rightBtn</tabstop>
  <tabstop>_leftBtn</tabstop>
  <tabstop>_projectA</tabstop>
  <tabstop>_projectB</tabstop>
 </tabstops>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
